sceptre/scipool/templates/sc-enduser-iam.yaml

```diff
@@ -57,7 +57,25 @@ Resources:
               - organizations:DescribeOrganization
               - servicecatalog:DescribeProvisionedProduct
               - cloudwatch:GetMetricData  # access to EC2 monitoring info
+              - ec2:DescribeImages  # service catalog status info
+              - lambda:ListTags  # service catalog tag updates
             Resource: "*"
+          - Effect: Allow
+            Action:
+              - "ec2:*Tags"
+              - "s3:*Tags"
```

Comment:
hmm, a service catalog product template can create multiple AWS resources and all of the resources are tagged when the product is provisioned, correct?  Therefore   Should this maybe be `*:*Tags` instead?  Also don't forget about the [Scheduled Jobs](https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/templates/batch/sc-batch-fargate.yaml) product.